frontend/src/components/DonationPicker.js

```diff
@@ -70,20 +70,26 @@ function customField({onChange, value, frequency, currency, showCurrencyPicker,
 
   var currencies = [{
     label: 'USD',
-    value: 'USD'
+    value: 'USD',
+    symbol: '$'
   }, {
     label: 'EUR',
-    value: 'EUR'
+    value: 'EUR',
+    symbol: '€'
   }, {
     label: 'MXN',
-    value: 'MXN'
+    value: 'MXN',
+    symbol: '$'
```

Comment:
This is fine when we show the currency picker (it's almost never though)

We need to tie the currency symbol to the group.currency. `format_currency` is likely the best place for it.